management-system/management-system/app/components/sidebar.jsx

```diff
@@ -0,0 +1,135 @@
+'use client';
+
+import Link from 'next/link';
+import { usePathname } from 'next/navigation';
+import { LuBriefcase } from 'react-icons/lu';
+import { LuUsers } from 'react-icons/lu';
+import { MdAdd } from 'react-icons/md';
+
+export default function Sidebar() {
+  const pathname = usePathname();
+
+  return (
+    <div className="flex bg-[#f9fafb] flex-col max-w-[255px] p-2 w-full gap-4 h-screen">
+      <div className="flex flex-col gap-6">
+        <div className="flex flex-col">
+          <h2 className="text-xl text-color-black font-extrabold px-5 py-3">
+            Management System
+          </h2>
+        </div>
+
+        <div className="flex flex-1 flex-col gap-1 overflow-auto">
+          <div className="h-6 text-[#3f3f46b2] px-2 font-semibold text-xs flex items-center outline-none">
+            Menu
+          </div>
+          <ul className="flex flex-col">
+            <li
+              className={`flex items-center cursor-pointer gap-4 text-sm p-2 rounded ${
+                pathname === '/employees' ? 'bg-gray-100' : 'hover:bg-gray-100'
```

Comment:
Shouldn’t the hover background color always be added, rather than conditionally? The code you’ve added is that when you’re on the employees route, it adds a bg-gray background, and when you’re not, it adds a hover background.